fbchat/graphql.py

```diff
@@ -172,6 +172,26 @@ def graphql_to_user(user):
         message_count=user.get('messages_count')
     )
 
+def graphql_to_thread(thread):
+    if thread['thread_type'] == 'GROUP':
+        return graphql_to_group(thread)
+    elif thread['thread_type'] == 'ONE_TO_ONE':
+        if thread.get('image') is None:
+            thread['image'] = {}
+        c_info = get_customization_info(thread)
```

Comment:
I think that around here you must have meant to say something like `return graphql_to_user(thread)`?

Reply:
You're right if "thread_type" is "ONE_TO_ONE" you should get a User not a Group.
But you can't just `return graphql_to_user(thread)` as the format is slightly different (it's more like a Group).

Response example:
```
{
  "thread_key": {
    "thread_fbid": null,
    "other_user_id": "yyyyyyyyy"
  },
  "name": null,
  "last_message": {
    
  },
  "messages_count": 327,
  "customization_info": null,
  "all_participants": {
    "nodes": [
      {
        "messaging_actor": {
          "id": "yyyyyyyyy",
          "__typename": "User",
          "name": "xxxxxxxxxx",
          "gender": "FEMALE",
          "url": "xxxxxxxxxx",
          "big_image_src": {
            "uri": "xxxxxxxxxx"
          },
          "short_name": "xxxxxxxxxx",
          "username": "xxxxxxxxxx",
          "is_viewer_friend": true,
          "is_messenger_user": true,
          "is_verified": false,
          "is_message_blocked_by_viewer": false,
          "is_viewer_coworker": false,
          "is_employee": null
        }
      },
      {
        "messaging_actor": {
          "id": "xxxxxxxxxx",
          "__typename": "User",
          "name": "xxxxxxxxxx",
          "gender": "MALE",
          "url": "xxxxxxxxxx",
          "big_image_src": {
            "uri": "xxxxxxxxxx"
          },
          "short_name": "xxxxxxxxxx",
          "username": "xxxxxxxxxx",
          "is_viewer_friend": false,
          "is_messenger_user": true,
          "is_verified": false,
          "is_message_blocked_by_viewer": false,
          "is_viewer_coworker": false,
          "is_employee": null
        }
      }
    ]
  }
}
```

I'm working on it :)

Reply:
Hmm, that's annoying ;). Thanks for the contribution, I'll probably merge it after testing it a bit